packages/zowe-explorer/package.json

```diff
@@ -1843,6 +1843,18 @@
           "description": "%zowe.settings.version%",
           "scope": "window"
         },
+        "zowe.settings.oldSettingsMigrated": {
+          "default": false,
+          "type": "boolean",
+          "description": "%zowe.settings.migrated%",
+          "scope": "window"
+        },
+        "zowe.settings.localStorageMigrated": {
+          "default": false,
+          "type": "boolean",
+          "description": "%zowe.settings.migrated%",
+          "scope": "window"
+        },
```

Comment:
What about these settings? They also seem like candidates for being migrated to local storage.
* `zowe.settings.version`
* `zowe.settings.oldSettingsMigrated`
* `zowe.settings.localStorageMigrated`

Reply:
These are being migrated directly to local storage initially since they haven't existed in ZE before and are new. `zowe.settings.version` needs to be removed since the second one in your list is replacing that, i'll go ahead and push a commit that removed the setting from `package.json` since it is no longer in use.